requirements-dev.txt

```diff
@@ -61,3 +61,4 @@ urllib3==1.26.2
 wcwidth==0.2.5
 wrapt==1.12.1
 zipp==3.4.0
+cached_property==1.5.2
```

Comment:
I found this on `setup.py` , but was missing in `requirements.txt`, so adding here for consistency (was required for local testing in pipenv)